t/Path.t

```diff
@@ -800,7 +800,11 @@ is(
         "If not provided with any paths, make_path() will return a count of 0 things created");
 }
 
-{
+ SKIP: {
+    my $skip_count = 5;
+    skip "getpwuid() and getgrgid() not implemented on Windows", $skip_count
+        if $^O eq 'MSWin32';
+
```

Comment:
I ended up using some aspects of your p.r. in version 2.12_005, which has now been released to CPAN.  Please give it a spin on your Strawberry Perl setup.

Thank you very much.
Jim Keenan